package.json

```diff
@@ -34,6 +39,7 @@
   "scripts": {
     "lint": "make lint",
     "pretest": "npm run lint",
-    "test": "mocha"
+    "test": "mocha",
+    "clientize": "webpack"
```

Comment:
If you make this `"webpack --optimize-minimize --optimize-dedupe"` you'll save about 300KB in the output.